src/commands/build.ts

```diff
@@ -43,6 +44,7 @@ try {
     sandbox: options.sandbox,
     executableChromium: options.executableChromium,
     image: options.image,
+    http: options.http,
```

Comment:
build コマンドで `--viewer` の指定が無視されています。
`viewer: options.viewer` を `build()` に渡すのを忘れているようです。

```suggestion
    http: options.http,
    viewer: options.viewer,
```